ios/VizbeeCastButtonView.m

```diff
@@ -33,6 +35,17 @@ - (void)setTintColor:(NSString *)tintColor {
     _tintColor = tintColor;
 }
 
+-(void)setDisabled:(BOOL)disabled{
+    if(self.castButton){
+        [self.castButton setEnabled:!disabled];
+    }
+    _disabled = disabled;
+}
+
+-(void)simulateButtonClick{
+    [self.castButton sendActionsForControlEvents:UIControlEventAllTouchEvents];
```

Comment:
@Sidharth-iOS Where are we checking for cast button visibility likewise in Android?
cc @shivareddy

Reply:
Done